Makefile

```diff
@@ -250,82 +265,89 @@ cover: $(COVER_ROOT)/cover.out
 cover_ci: $(COVER_ROOT)/cover.out
 	goveralls -coverprofile=$(COVER_ROOT)/cover.out -service=buildkite || echo Coveralls failed;
 
-clean:
```

Comment:
wondering if `clean` would be a good target to preserve (even if all it does is include all other `clean-*` targets).

Reply:
yeah, good point. I'll add it to the top ones.